### ISSUE TITLE
Generate a cfwheels API docset for Dash application

### ISSUE THREAD

**Issue by chapmandu** [wheels-dev/wheels#1133]
Generate a cfwheels API docset for 
- https://kapeli.com/dash (MacOS)
- https://zealdocs.org/ (Linux/Windows)

Auto-generate the documents as outlined https://kapeli.com/docsets 

There is a [vscode](https://github.com/deerawan/vscode-dash) extension that can integrate any locally installed Dash docset. It supports snippets and API lookup shortcuts.

CFWheels already has methods to parse its own Javadocs and a [JSON endpoint](https://api.cfwheels.org/files/json/v2.2.json) containing all the API docs.

I have used it to generate a [vscode snippets extension](https://github.com/chapmandu/cfwheels-code-snippets/blob/master/src/build.cfm)

**Comment by bpamiri** [comment]
I love this idea. ColdFusion is listed as one of the supported documentation sets. I also saw Ruby listed as well as Ruby on Rails. Looking at the docsets link it seems they support Javadoc along with a bunch of other input formats. So is your idea to either take our json generated doc file as an input to one of their parsers or somehow create an export that one of their parsers can read to generate the docset. There also seemed to be a way to craft a docset in HTML and folders.

This could be added to the build process so the docset could get updated automatically on each push. (Something we don’t currently do for the API docs, that process is manual at the moment).

**Comment by chapmandu** [comment]
Yes, I think the idea would be take our JSON generated doc file as an input to one of their parsers. It may also be possible to use the Guides markdown to generate HTML pages.. similar to Rails docs.

**Comment by zainforbjs** [comment]
@chapmandu Instead of generating a Dash/Zeal docset, we’ve developed a dedicated **[VS Code extension](https://marketplace.visualstudio.com/items?itemName=wheels-dev.wheels-vscode)** that provides:

* API documentation
* Intelligent snippets
* Quick lookup support for CFWheels

This makes it much easier to access and use documentation directly within the editor.
CC: @bpamiri 

Closing this issue as **resolved**.
